examples/auth.rs

```diff
@@ -14,9 +14,9 @@ async fn main() -> Result<()> {
         .await
         .unwrap();
 
-    session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'replication_factor' : 1}", &[]).await.unwrap();
+    session.query("CREATE KEYSPACE IF NOT EXISTS examples_ks WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'replication_factor' : 1}", &[]).await.unwrap();
     session
-        .query("DROP TABLE IF EXISTS ks.t;", &[])
+        .query("DROP TABLE IF EXISTS examples_ks.auth;", &[])
```

Comment:
Hmm... Previously thix example created (if not exists) the same keyspace as other tests, and then dropped (again, conditionally) shared table.
Now the table is not shared, so no other example can create it, dropping it makes not much sense.
Maybe to verify that session really works, let's remove keyspace / table creation / deletion and just perform some query like `SELECT * FROM system.local`?
wdyt @piodul

Reply:
Alternatively, create a table and insert some data into it

Reply:
An example can be run more than once and might expect the table to be in an empty state. I think it still makes sense to drop the table at the beginning.

Reply:
What I meant is that this example does 2 queries: CREATE IF NOT EXISTS and then DROP IF EXISTS - which doesn't really showcase that session actually works, like inserting / selecting / printing data would.
But it's hard for me to imagine scenario where those queries don't return error in a non-functioning session, so I guess it's ok.

Reply:
I agree that the statements that this sample executes are weird, but IIUC they do test that the session works - if the credentials were wrong, then we wouldn't be able to establish any connection to the cluster and wouldn't be able to send anything. We are using superuser credentials, so this example doesn't test authorization but rather authentication - we might consider extending it so that it creates some users and verifies that permissions work.

In any case, those issues are out of scope of the PR.